From 3479eaed36a32c6bd91331573a450a5bf099bdd5 Mon Sep 17 00:00:00 2001
From: Alex Gaynor <alex.gaynor@gmail.com>
Date: Mon, 1 Jul 2019 00:11:38 -0400
Subject: [PATCH] Fixed tests for change in pytest repr (#546)

* Fixed tests for change in pytest repr

* Fixed tests for change in pytest repr

* Fixed tests for change in pytest repr

* Fixed tests for change in pytest repr
---
 tests/test_box.py        | 2 +-
 tests/test_sealed_box.py | 2 +-
 tests/test_secret.py     | 2 +-
 tests/test_signing.py    | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/tests/test_box.py b/tests/test_box.py
index d547ac27..46f3b793 100644
--- a/tests/test_box.py
+++ b/tests/test_box.py
@@ -268,7 +268,7 @@ def test_box_wrong_length():
 def check_type_error(expected, f, *args):
     with pytest.raises(TypeError) as e:
         f(*args)
-    assert expected in str(e)
+    assert expected in str(e.value)
 
 
 def test_wrong_types():
diff --git a/tests/test_sealed_box.py b/tests/test_sealed_box.py
index f7ba15c3..49ea2324 100644
--- a/tests/test_sealed_box.py
+++ b/tests/test_sealed_box.py
@@ -110,7 +110,7 @@ def test_sealed_box_decryption(privalice, pubalice, plaintext, encrypted):
 def check_type_error(expected, f, *args):
     with pytest.raises(TypeError) as e:
         f(*args)
-    assert expected in str(e)
+    assert expected in str(e.value)
 
 
 def test_wrong_types():
diff --git a/tests/test_secret.py b/tests/test_secret.py
index 3b52459a..73987fcb 100644
--- a/tests/test_secret.py
+++ b/tests/test_secret.py
@@ -142,7 +142,7 @@ def test_secret_box_wrong_lengths():
 def check_type_error(expected, f, *args):
     with pytest.raises(TypeError) as e:
         f(*args)
-    assert expected in str(e)
+    assert expected in str(e.value)
 
 
 def test_wrong_types():
diff --git a/tests/test_signing.py b/tests/test_signing.py
index 4304afb9..3e2cf359 100644
--- a/tests/test_signing.py
+++ b/tests/test_signing.py
@@ -219,7 +219,7 @@ def test_key_conversion(self):
 def check_type_error(expected, f, *args):
     with pytest.raises(TypeError) as e:
         f(*args)
-    assert expected in str(e)
+    assert expected in str(e.value)
 
 
 def test_wrong_types():
From a8c08b18f3a2e8f2140c531afaf42715fcab68e7 Mon Sep 17 00:00:00 2001
From: Valentin Heidelberger <31535155+va1entin@users.noreply.github.com>
Date: Fri, 28 Sep 2018 00:05:06 +0200
Subject: [PATCH] Remove average_size hypothesis setting from test_bindings.py
 (#480)

---
 tests/test_bindings.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tests/test_bindings.py b/tests/test_bindings.py
index 22930cc7..d7951a21 100644
--- a/tests/test_bindings.py
+++ b/tests/test_bindings.py
@@ -306,7 +306,6 @@ def test_unpad_not_padded():
 
 
 @given(binary(min_size=0,
-              average_size=128,
               max_size=2049),
        integers(min_value=16,
                 max_value=256)
@@ -320,7 +319,6 @@ def test_pad_sizes(msg, bl_sz):
 
 
 @given(binary(min_size=0,
-              average_size=128,
               max_size=2049),
        integers(min_value=16,
                 max_value=256)
